tests/Oryx.Integration.Tests/Python/PythonJamSpellAppTests.cs

```diff
@@ -0,0 +1,110 @@
+﻿// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
+
+using Microsoft.Oryx.Common;
+using Microsoft.Oryx.Tests.Common;
+using System.Threading.Tasks;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.Oryx.Integration.Tests
+{
+    [Trait("category", "python")]
+    public class PythonJamSpellAppTests : PythonEndToEndTestsBase
+    {
+        public PythonJamSpellAppTests(ITestOutputHelper output, TestTempDirTestFixture fixture)
+            : base(output, fixture)
+        {
+        }
+
+        [Theory(Skip = "locale bug, https://github.com/bakwc/JamSpell/issues/17")]
+        [MemberData(nameof(TestValueGenerator.GetPythonVersions), MemberType = typeof(TestValueGenerator))]
+        public async Task CanBuildAndRun_JamSpellFlaskApp_UsingVirtualEnv(string pythonVersion)
+        {
+            // Arrange
+            var appName = "jamspell-flask-app";
+            var volume = CreateAppVolume(appName);
+            var appDir = volume.ContainerDir;
+            var buildScript = new ShellScriptBuilder()
+               .AddCommand($"oryx build {appDir} --platform python --platform-version {pythonVersion}")
+               .ToString();
+            var runScript = new ShellScriptBuilder()
+                .AddCommand($"oryx -appPath {appDir} -bindPort {ContainerPort}")
+                .AddCommand(DefaultStartupFilePath)
```

Comment:
As discussed let's see if setting the locale related environment variables here (simulating how an AppService end user would can do it through app settings) fixes the issue